config/application.rb

```diff
@@ -41,6 +41,9 @@ class Application < Rails::Application
     # User will be timed out after 15 minutes of inactivity
     config.session_inactivity = 15.minutes
 
+    # The cache reload time for the JWKS file from amazon
+    config.jwks_expiry = 1.hour
+
     config.filter_parameters << :temporary_password << :new_password << :password
```

Comment:
Does the JWT get spat out in any of the logs? If so it would be good to filter it in `config.filter_parameters` like the passwords are.

Reply:
No this isn't put in any of the logs nor is the raw JWT ever shown